src/UserSpaceInstrumentation/RegisterStateTest.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+#include <stdint.h>
+#include <sys/ptrace.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+
+#include "OrbitBase/Logging.h"
+#include "UserSpaceInstrumentation/RegisterState.h"
+
+namespace orbit_user_space_instrumentation {
+
+namespace {
+
+// Let the parent trace us, write into rax and ymm0, then break. While the child is breaked the
```

Comment:
What do you mean with "break" in this context?

Reply:
Entering a breakpoint. The int3 in the asm.  Let's chat about that offline as well ...

Reply:
done